### PR TITLE
SignInAPITests

### PR DESCRIPTION
**Pull request by Hotavya**

### CODE REVIEW

File: server/test/login.test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                res.should.have.status(200);
+                expect(res.body.error).to.be.equal(undefined);
+                expect(res.body.token).to.not.be.null;
+                console.log('Res body message: ', res.body.message);
+                console.log('Token: ', res.body.token);
+                done();
+            })
+    })
+
+    it("Should not login user and return a status code of 500", (done) => {
```
Comment (JoeKabongo) [inline server/test/login.test.js:29]: I would branch this out in two cases. "Should not login when user enter an email that does not exist in the system" and "Should not login when password is not correct"

**Review by bmahanloo — APPROVED** [review]
LGTM

File: server/test/login.test.js (hunk 1 above)
Comment (Hotavya) [reply server/test/login.test.js]: Added the requested changes

**Comment by Hotavya** [comment]
lgtm

**Pull request merged**